### PR TITLE
Release/1.1.1

### PR DESCRIPTION
**Pull request by troydieter**
Added token lease duration, along with the leaseID which is valuable for the token requester.

### CODE REVIEW

**Comment by troydieter** [comment]
Can you assist with possibly using the `TimeSpan` class to convert the time from seconds? I would imagine it'd be a more presentable format.

File: src/VaultCredentialRotator/AwsProvider.cs [hunk 1]
```diff
@@ -100,10 +99,16 @@ public async Task ExecuteAsync(IConfigurationRoot configuration)
...
             var secretKey = (string)awsCredsDeserializedObject.data.secret_key;
+            var leaseID = (string)awsCredsDeserializedObject.lease_id;
+            var leaseDuration = (string)awsCredsDeserializedObject.lease_duration;
 
             Console.WriteLine();
             await SaveCredentialsAsync(accessKey, secretKey);
             Console.WriteLine("Credentials saved!");
+            Console.ForegroundColor = ConsoleColor.Yellow;
+            Console.WriteLine("Vault Lease ID: " + leaseID);
+            Console.WriteLine("Vault Token Lease Duration: " + leaseDuration + " seconds");
```
Comment (issafram) [inline src/VaultCredentialRotator/AwsProvider.cs]: Change to use [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated)

**Comment by issafram** [comment]
> Can you assist with possibly using the `TimeSpan` class to convert the time from seconds? I would imagine it'd be a more presentable format.

Convert it to what? What output are you looking for?

**Comment by troydieter** [comment]
> > Can you assist with possibly using the `TimeSpan` class to convert the time from seconds? I would imagine it'd be a more presentable format.
> 
> Convert it to what? What output are you looking for?

Referring to something like this: https://docs.microsoft.com/en-us/dotnet/api/system.timespan?redirectedfrom=MSDN&view=netframework-4.8

**Pull request merged**